### ISSUE TITLE
Link to code browser

### ISSUE THREAD

**Issue by bloodearnest** [opensafely-core/airlock#267]
When #238 and https://github.com/opensafely-core/job-runner/issues/701 have landed, we can provide direct links from a file you a viewing to the commit that produced it.

This is probably just another file-level button, that links to the code browser at the right commit.  It should probably open in a new tab?

**Comment by bloodearnest** [comment]
Assuming that the commit for a specific output file will be stored in the `metadata/manifest.json`, as per current plan.

This information is not needed for the nav tree, so it would probably be best to defer looking up the commit until the link is clicked on.

One option would be to only load the commit data from manifest.json when rendering the button for a specific file.  In the same we way we have `get_contents_url()`, we could have `get_code_url()` on the ReleaseRequest object, and use it to supply the link href.

**Comment by rebkwok** [comment]
No longer blocked on #303, but is still blocked on https://github.com/opensafely-core/job-runner/issues/729
